pylint/checkers/imports.py

```diff
@@ -800,10 +801,10 @@ def _get_imported_module(
                 return None
             self.add_message("relative-beyond-top-level", node=importnode)
         except astroid.AstroidSyntaxError as exc:
-            message = (
-                f"Cannot import {modname!r} due to syntax error {str(exc.error)!r}"
+            message = f"Cannot import {modname!r} due to {exc.error}"
```

Comment:
Is `!r` necessary?

Reply:
I discovered it's an easy way to surround strings by quote (ie it's f"'{modename}'").